src/models/payments.js

```diff
@@ -316,13 +313,13 @@ function * settle (payment) {
   }
 }
 
-function isTraderFunds (funds) {
+function isTraderFunds (config, funds) {
```

Comment:
Any reason why `config` is the first param here. Above it's towards the end of the parameter list.